### PR TITLE
Modinvstor 381 v01 Store Shelving Order in item

### PR DESCRIPTION
**Pull request by AnShapovalov**

### CODE REVIEW

**Review by oleksandr-yatsenko1 — APPROVED** [review]
Looks good for me. Thanks for resolving all the concerns.

File: src/test/java/org/folio/rest/api/ItemStorageTest.java [hunk 1]
```diff
@@ -117,6 +120,85 @@ public void checkIdsAfterEach() {
...
+    "PN 12 A69 V 11 NO 11 41921,PN12 .A69 V.11 NO.11 41921,,PN2 .A69,v.1,no. 1,1921,,",
+    "PR 49199.3 41920 L33 41475 A6,PR 49199.3 41920 .L33 41475 .A6,,PR9199.3 1920 .L33 1475 .A6,,,,,",
+    "PQ 42678 K26 P54,PQ 42678 .K26 P54,,PQ2678.K26 P54,,,,,",
+    "PQ 48550.21 R57 V5 41992,PQ 48550.21 .R57 V15 41992,,PQ8550.21.R57 V5 1992,,,,,",
+    "PQ 48550.21 R57 V5 41992,PQ 48550.21 .R57 V15 41992,,PQ8550.21.R57 V5,,,1992,,",
+    "PR 3919 L33 41990,PR 3919 .L33 41990,,PR919 .L33 1990,,,,,",
+    "PR 49199 A39,PR 49199 .A39,,PR9199 .A39,,,,,",
+    "PR 49199.48 B3,PR 49199.48 .B3,,PR9199.48 .B3,,,,,"
+  })
+  @Test
```
Comment (marcjohnson-kint) [inline src/test/java/org/folio/rest/api/ItemStorageTest.java:144]: Why does this API test need to be executed many times when the variation in the process is already reflected in the `CallNumberUtilsTest` class?

File: src/test/java/org/folio/rest/api/HoldingsStorageTest.java [hunk 2]
```diff
@@ -600,6 +605,129 @@ public void tenantIsRequiredForGettingAllHoldings() throws InterruptedException,
...
+    "PN 12 A69 V 11 NO 11 +,PN12 .A69 V.11 NO.11 +,Over,PN2 .A69,v.1,no. 1,,,+",
+    "PN 12 A69 V 11 NO 11 41921,PN12 .A69 V.11 NO.11 41921,,PN2 .A69,v.1,no. 1,1921,,",
+    "PR 49199.3 41920 L33 41475 A6,PR 49199.3 41920 .L33 41475 .A6,,PR9199.3 1920 .L33 1475 .A6,,,,,",
+    "PQ 42678 K26 P54,PQ 42678 .K26 P54,,PQ2678.K26 P54,,,,,",
+    "PQ 48550.21 R57 V5 41992,PQ 48550.21 .R57 V15 41992,,PQ8550.21.R57 V5 1992,,,,,",
+    "PQ 48550.21 R57 V5 41992,PQ 48550.21 .R57 V15 41992,,PQ8550.21.R57 V5,,,1992,,",
+    "PR 3919 L33 41990,PR 3919 .L33 41990,,PR919 .L33 1990,,,,,",
+    "PR 49199 A39,PR 49199 .A39,,PR9199 .A39,,,,,",
+    "PR 49199.48 B3,PR 49199.48 .B3,,PR9199.48 .B3,,,,,"
+  })
```
Comment (marcjohnson-kint) [inline src/test/java/org/folio/rest/api/HoldingsStorageTest.java:628]: Why does this API test need to be executed many times when the variation in the process is already reflected in the `CallNumberUtilsTest` class?

File: src/test/java/org/folio/rest/api/StorageTestSuite.java [hunk 3]
```diff
@@ -40,6 +42,7 @@
 
 @RunWith(Suite.class)
 @Suite.SuiteClasses({
+  CallNumberUtilsTest.class,
```
Comment (marcjohnson-kint) [inline src/test/java/org/folio/rest/api/StorageTestSuite.java:45]: Why is this test class included in the suite when it doesn't need infrastructure?

**Pull request merged**